client/app/billing/autoRenew/actions/activation/activation.controller.js

```diff
@@ -2,11 +2,17 @@ import _ from 'lodash';
 
 export default class {
   /* @ngInject */
-  constructor($translate) {
+  constructor($translate, atInternet) {
```

Comment:
```suggestion
  constructor(
    $translate,
    atInternet,
  ) {
```
makes it easier to inject new dependencie ;)